e2e/app-with-domain/app_with_domain_test.go

```diff
@@ -298,6 +301,19 @@ var _ = Describe("App With Domain", func() {
 					resp, fetchErr = http.Get(strings.Replace(url, "https", "http", 1))
 					return resp.StatusCode, fetchErr
 				}, "60s", "1s").Should(Equal(200))
+				Eventually(func() (int, error) {
+					resp, fetchErr = http.Get(fmt.Sprintf("%s/admin", strings.Replace(url, "https", "http", 1)))
+					body, err := io.ReadAll(resp.Body)
+					if err != nil {
+						return 0, err
+					}
+					fmt.Printf("response: %s\n", string(body))
```

Comment:
remove prints for cleaner logs? it will be in the error message if it's unexpected